src/Boards.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from "react";
+import "./App.css";
+import Tasks from "./components/Tasks.jsx";
+import axios from "axios";
+import { connect } from 'react-redux';
+
+class Boards extends Component {
+  constructor(props) {
+    super(props);
+
+  }
+
+
+  render() {
+    // console.log("what is the state?: ", this.props);
+    const daBoard = this.props.boards.boardsName;
+    const daTask = this.props.tasks
+    // console.log("CHECK CHECK", daBoard);
+    return (
+      
+      // adding cards based on title of the board
+      <div className="Boards">
+        {daBoard.map(item => {
+          // console.log("Board", item);
+          if (item.title === "Done") {
+            return (
+              <div className="Categories" key={item.id}>
+                {item.title}
+
+                <DoneCards tasks={daTask} />
+              </div>
+            );
+          } else if (item.title === "In Progress"){
+            return (
+              <div className="Categories" key={item.id}>
+                {item.title}
+
+                <ProgressCards tasks={daTask} />
+              </div>
+            );
+          } else if (item.title === "In Queue"){
+            return (
+              <div className="Categories" key={item.id}>
+                {item.title}
+
+                <QueueCards tasks={daTask} />
+              </div>
+            );
+          }
+        })}
+      </div>
+    );
+  }
+}
+
+
+// filter for cards with a In Progress status
+function ProgressCards(props) {
+  const filteredArr = props.tasks.filter(tasks => {
+    return tasks.status === "In Progress";
+  });
+  // console.log("In Progress FILTER", filteredArr);
+  return filteredArr.map(tasks => {
+    return (
+      <Tasks
+        Cards
+        key={tasks.tasks_id}
+        tasks_id={tasks.tasks_id}
+        title={tasks.title}
+        details={tasks.details}
+        priority={tasks.priority}
+        status={tasks.status}
+        assignedTo_UserID={tasks.assignedTo_UserID}
+        createdBy_UserID={tasks.createdBy_UserID} 
+      />
+    );
+  });
+}
+
+// filter for cards with a In Queue status
+function QueueCards(props) {
+  // console.log("CAAARRRDS", props);
+  const filteredArr = props.tasks.filter(tasks => {
+    return tasks.status === "In Queue";
+  });
+  // console.log("In Queue FILTER", filteredArr);
+  return filteredArr.map(tasks => {
+    return (
```

Comment:
A common pattern is to destructure like this before putting into props:
```suggestion
    const { task_id, title, details, priority, status, assignedTo_UserID, createdBy_UserID } = tasks
```